src/fusion-order/settlement-post-interaction-data/settlement-post-interaction-data.ts

```diff
@@ -156,8 +169,13 @@ export class SettlementPostInteractionData {
         if (this.integratorFee?.ratio) {
```

Comment:
`&& !this.integratorFee.receiver.isZero()`, because someone can forget to provide `takigFeeReceiver` here: https://github.com/1inch/limit-order-settlement-lib/blob/ff6b6d5bf6398f482b3fc2c96f2c357c86bcecc5/src/api/quoter/quote/quote.ts#L94

Or better to throw an error in case `ratio` non zero and `receiver` is zero